### DISCUSSION TITLE
Missing Plugin 1/0 Layouts

### DISCUSSION THREAD

**Discussion by mwambasiri** [apohl79/audiogridder#1061] [Help] [answered]
hi,
always get this error when launch:
Missing Plugin 1/0 Layouts
No cached plugin layouts have been found. This can increase plugin loading times
and not all 1/O configurations might be available.
DO you want to rescan your plugins?
i'm on osx big sur

**Comment by apohl79** [comment, marked as answer]
I assume that the rescan does not work. Have you tried to "wipe and rescan"? You can also try to delete the plugin cache (see https://github.com/apohl79/audiogridder/discussions/1024#discussioncomment-3635291). Please provide the server logs, if you can't resolve this.

**Comment by mwambasiri** [comment]
i did wipe & rescan without success, i try the command ~/.audiogridder/audiogridderserver.cfg in terminal i get
.audiogridder/audiogridderserver.cfg: Not a directory
i also notice that when i make changes in settings, the changes are not saved
how can i get the server logs?

**Reply by apohl79** [reply]
To wipe all AG settings, you have to run `rm -rf ~/.audiogridder` on macOS. The server logs can be found here: https://audiogridder.com/bug-report-diagnostics-locations/

**Comment by mwambasiri** [comment]
rm -rf ~/.audiogridder solves everything.
thank's for your help!
AG is a really promising plugin, good job guys!
